setup.py

```diff
@@ -0,0 +1,29 @@
+# coding: utf-8
+
+from setuptools import setup, find_packages  # noqa: H301
+
+
+setup(
+    name='fasjsonclient',
+    version='0.0.1',
+    description='Fedora Account Service JSON API Client',
+    author='Fedora Inrastucture team',
+    author_email='team@openapitools.org',
+    url='https://github.com/fedora-infra/fasjson-client',
+    keywords=['OpenAPI', 'Fedora Account Service JSON API', 'fas'],
+    install_requires=[
+        'urllib3 >= 1.25.8',
+        'six >= 1.14.0',
+        'certifi==2019.11.28',
+        'python-dateutil==2.8.1',
+        'gssapi==1.6.2',
+        'requests==2.23.0',
+        'requests-gssapi==1.2.0',
```

Comment:
This is already defined in `requirements.txt`, no? We should probably import that file here.